dicompylercore/dose.py

```diff
@@ -0,0 +1,283 @@
+        return [self.x_axis, self.y_axis, self.z_axis]
+
+    @property
+    def scale(self):
+        """Get the dose grid resolution (xyz)"""
+        return np.array([self.ds.PixelSpacing[0],
+                         self.ds.PixelSpacing[1],
+                         self.ds.GridFrameOffsetVector[1] - self.ds.GridFrameOffsetVector[0]])
+
+    @property
+    def offset(self):
+        """Get the coordinates of the dose grid origin (mm)"""
+        return np.array(self.ds.ImagePositionPatient, dtype='float')
+
+    @property
+    def points(self):
+        """Get all of the points in the dose grid"""
+        y, x, z = np.meshgrid(self.y_axis, self.x_axis, self.z_axis)
+        points = np.vstack((x.ravel(), y.ravel(), z.ravel()))
+        return points.transpose()
+
+    ####################################################
+    # Tools
+    ####################################################
+    def __add__(self, other):
+        """Addition in this fashion will not alter either DoseGrid, but it is more expensive with memory"""
+        new = deepcopy(self)
+        new.add(other)
+        return new
+
+    def is_coincident(self, other):
+        """Check dose grid coincidence, if True a direct summation is appropriate"""
+        return self.ds.ImagePositionPatient == other.ds.ImagePositionPatient and \
+               self.ds.pixel_array.shape == other.ds.pixel_array.shape and \
+               self.ds.PixelSpacing == other.ds.PixelSpacing and \
+               self.ds.GridFrameOffsetVector == other.ds.GridFrameOffsetVector
+
+    def set_pixel_data(self):
+        """
+        Update the PixelData in the pydicom.FileDataset with the current self.dose_grid
+        """
+        self.ds.BitsAllocated = 32
+        self.ds.BitsStored = 32
+        self.ds.HighBit = 31
+        self.ds.DoseGridScaling = np.max(self.dose_grid) / np.iinfo(np.uint32).max
+        pixel_data = np.swapaxes(self.dose_grid, 0, 2) / self.ds.DoseGridScaling
+        self.ds.PixelData = np.uint32(pixel_data).tostring()
+
+    def save_dcm(self, file_path):
+        """Save the pydicom.FileDataset to file"""
+        self.ds.save_as(file_path)
+
+    def get_ijk_points(self, other_axes):
+        """
+        Convert axes from another DoseGrid into ijk of this DoseGrid
+        :param other_axes: the x, y, and z axis arrays
+        :type other_axes: list
+        :return: np.vstack of other_axes in this ijk space
+        """
+        ijk_axes = [(np.array(axis) - self.offset[a]) / self.scale[a] for a, axis in enumerate(other_axes)]
+        j, i, k = np.meshgrid(ijk_axes[1], ijk_axes[0], ijk_axes[2])
+        return np.vstack((i.ravel(), j.ravel(), k.ravel()))
+
+    ####################################################
+    # Dose Summation
+    ####################################################
+    def add(self, other):
+        """
+        Add another 3D dose grid to this 3D dose grid, with interpolation if needed
```

Comment:
I cloned your repo to start building the docs and saw that the new module, i.e. `dvh` and `dvhcalc` we are using Google formatting for docstrings. This formatting looks to be in ReST format.